resolvers/webpack/README.md

```diff
@@ -25,3 +27,13 @@ settings:
   import/resolver:
     webpack: { config: 'webpack.dev.config.js' }
 ```
+
+or with explicit config file name:
+
+```yaml
+---
+settings:
+  import/resolver:
+    webpack: { config: 'webpack.multiple.config.js' }
+    config-index: 1
```

Comment:
This actually needs to be a sibling of the "config" key, I think. The settings that the resolver gets are the contents of the object keyed by "webpack"